bin/docker-build-ns-labeler

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
```

Comment:
I avoided adding it so that `bin/image-load` doesn't attempt to import it using the tag given by `head_root_tag` instead of the fixed one. But given my comment above, I think I'll add it and have `bin/docker-build-ns-labeler` use the dynamic tags as the other images. The tag will get overriden via a `TAG` env var when releasing the `ns-labeler` image.

Reply:
Makes sense. Our CI is so tightly coupled to all helper scripts :(